jobbergate-api/jobbergate_api/apps/job_submissions/schemas.py

```diff
@@ -60,9 +60,218 @@
+    )
+    burst_buffer: Optional[str] = Field(description="Burst buffer specification.")
+    cluster_constraints: Optional[str] = Field(
+        description=(
+            "Specifies features that a federated cluster must have to have a sibling job submitted to it."
+        )
+    )
+    comment: Optional[str] = Field(description="An arbitrary comment.")
+    constraints: Optional[str] = Field(description="node features required by job.")
+    core_specification: Optional[int] = Field(
+        description=(
+            "Count of specialized threads per node reserved by the job for system "
+            "operations and not used by the application."
+        )
+    )
+    cores_per_socket: Optional[int] = Field(
+        description=(
+            "Restrict node selection to nodes with at least the specified number of cores per socket."
+        )
+    )
+    cpu_binding: Optional[str] = Field(description="Cpu binding")
+    cpu_binding_hint: Optional[str] = Field(description="Cpu binding hint")
+    cpu_frequency: Optional[str] = Field(
+        description=(
+            "Request that job steps initiated by srun commands inside this sbatch "
+            "script be run at some requested frequency if possible, on the CPUs "
+            "selected for the step on the compute node(s)."
+        ),
+    )
+    cpus_per_gpu: Optional[str] = Field(description="Number of CPUs requested per allocated GPU.")
+    cpus_per_task: Optional[int] = Field(
+        description=(
+            "Advise the Slurm controller that ensuing job steps will require "
+            "ncpus number of processors per task."
+        ),
+    )
+    current_working_directory: Optional[str] = Field(
+        description="Instruct Slurm to connect the batch script's standard output directly to the file name."
+    )
+    deadline: Optional[str] = Field(
+        description=(
+            "Remove the job if no ending is possible before this deadline (start > (deadline - time[-min]))."
+        )
+    )
+    delay_boot: Optional[int] = Field(
+        description=(
+            "Do not reboot nodes in order to satisfied this job's feature specification if "
+            "the job has been eligible to run for less than this time period."
+        )
+    )
+    dependency: Optional[str] = Field(
+        description=(
+            "Defer the start of this job until the specified dependencies have been satisfied completed."
+        )
+    )
+    distribution: Optional[str] = Field(
+        description="Specify alternate distribution methods for remote processes."
+    )
+    environment: Optional[Dict[str, str]] = Field(description="Dictionary of environment entries.")
+    exclusive: Optional[Literal["user", "mcs", "exclusive", "oversubscribe"]] = Field(
+        description=(
+            "The job allocation can share nodes just other users with the "
+            "'user' option or with the 'mcs' option)."
+        )
+    )
+    get_user_environment: Optional[bool] = Field(
```

Comment:
This one is `int` the cluster-agent, another point to keep an eye on integration tests:
https://github.com/omnivector-solutions/cluster-agent/blob/229b35ca804c60831ba3c9ba6ba4b7253773def4/cluster_agent/jobbergate/schemas.py#L46

Reply:
indeed is an int. I checked my internal notes and I verified it is an int. `1` or `0`.